doc/tools/make_rst.py

```diff
@@ -74,13 +75,23 @@
     "node": "Nodes",
     "resource": "Resources",
     "object": "Other objects",
+    "editor": "Editor-only",
     "variant": "Variant types",
 }
 CLASS_GROUPS_BASE: Dict[str, str] = {
     "node": "Node",
     "resource": "Resource",
     "object": "Object",
+    "variant": "Variant",
 }
+# Sync with editor\register_editor_types.cpp
+EDITOR_CLASSES: List[str] = [
+    "AnimationTrackEditPlugin",
```

Comment:
I noticed that this class is completely unusable and empty, so I made a PR to remove it: https://github.com/godotengine/godot/pull/76413. If it gets removed, this line also needs to be removed.

Reply:
This can be removed in #76413 itself now :)